packages/web3/README.md

```diff
@@ -26,4 +28,6 @@ All the TypeScript typings are placed in the `types` folder.
 [deps-image]: https://david-dm.org/ethereum/web3.js/1.x/status.svg?path=packages/web3
 [deps-url]: https://david-dm.org/ethereum/web3.js/1.x?path=packages/web3
 [deps-dev-image]: https://david-dm.org/ethereum/web3.js/1.x/dev-status.svg?path=packages/web3
-[deps-dev-url]: https://david-dm.org/ethereum/web3.js/1.x?type=dev&path=packages/web3
\ No newline at end of file
+[deps-dev-url]: https://david-dm.org/ethereum/web3.js/1.x?type=dev&path=packages/web3
```

Comment:
this link returns 504 `https://david-dm.org/ethereum/web3.js/1.x?type=dev&path=packages/web3`